es6-template-literals/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Javascript ES6 - Template Literals</title>
+  </head>
+  <body>
+    <script>
+      const number = 9200;
+      const street = `Irvine Center Drive`;
+      const address = number + ` ` + street;
```

Comment:
Please review the documentation: https://javascripttutorial.net/es6/javascript-template-literals/

Specifically the section **Variable and expression substitutions**